### PR TITLE
chore: move install button to the pagetools section

### PR DESCRIPTION
**Pull request by rudouglas**
Moved CTA to pagetools section
Responsive on Mobile

### CODE REVIEW

**Comment by rudouglas** [comment]
Screenshot 2021-08-31 at 15 07 20

File: src/templates/QuickstartDetails.js [hunk 1]
```diff
@@ -164,13 +167,34 @@ const QuickstartDetails = ({ data, location }) => {
...
+                  @media (max-width: 1240px) {
+                    justify-content: flex-start;
+                  }
+                  @media (max-width: 760px) {
+                    flex-direction: column;
+                    align-items: stretch;
+                  }
+                `}
+              >
+                <InstallButton quickstart={pack} onClick={handleInstallClick} />
```
Comment (jpvajda) [inline src/templates/QuickstartDetails.js:183]: These buttons look really far apart in the screenshots, can you look at the spacing we use on the Docs Site and adjust the margin v between the buttons so they are tighter? 

https://docs.newrelic.com/docs/using-new-relic/welcome-new-relic/get-started/find-help-use-support-portal/

**Review by jpvajda — CHANGES_REQUESTED** [review]

**Comment by jpvajda** [comment]
Are you also adding in the + icon from the feather library in this PR? See my comment on the issue, we do need that `+` icon

**Comment by rudouglas** [comment]
changes completed

**Review by aswanson-nr — APPROVED** [review]

**Comment by zstix** [comment]
Ugh 🤦  I caught an issue with this just _after_ I merged in this PR: the icon isn't being rendered.

I'll submit a follow-up PR to address this and some other tweaks.

**Pull request merged**